group-coordinator/src/main/java/org/apache/kafka/coordinator/group/consumer/ConsumerGroupMember.java

```diff
@@ -528,27 +478,6 @@ public Map<Uuid, Set<Integer>> partitionsPendingRevocation() {
         return partitionsPendingRevocation;
     }
 
-    /**
-     * @return The set of partitions awaiting assignment to the member.
-     */
-    public Map<Uuid, Set<Integer>> partitionsPendingAssignment() {
-        return partitionsPendingAssignment;
-    }
-
-    /**
-     * @return A string representation of the current assignment state.
-     */
-    public String currentAssignmentSummary() {
```

Comment:
Did we get rid of this because it is not used anywhere?

Reply:
Correct.